app/src/org/odk/collect/android/widgets/IntentWidget.java

```diff
@@ -84,7 +84,6 @@ public void makeTextView(FormEntryPrompt prompt) {
         }
 
         // finish complex layout
-        addView(launchIntentButton);
```

Comment:
If the bug is that this is null at startup, that also applies to the above condition (s != null, results in a direct dereference), that should also be cleaned up before this is accepted.